columnflow/tasks/framework/base.py

```diff
@@ -990,6 +990,9 @@ class ConfigTask(AnalysisTask):
         description=f"name of the analysis config to use; default: '{default_config}'",
     )
 
+    # the field in the store parts behind which the new part is inserted
+    store_part_anchor = "task_family"
```

Comment:
mhh, this would have been the point where I'd differentiate between "config" and "configs" (In `MultiConfigTask`), but now there is no unique definition of the `store_part_anchor. I'd prefer to keep this simple and only distinguish between "config" and "configs" using the classattribute

Also how does it work with class inheritance and having multiple definitions of `store_part_anchor`? Do we not just end up defining it as whatever is set by the first class in our inheritance? Is that what is intended?